resources/extra-large-conference/README.md

```diff
@@ -72,16 +80,35 @@ s2s_whitelist = {
 
 - Make sure s2s is not in modules_disabled
 - Enable `"xxl_conference";` module under the main virtual host (e.g. [here](https://github.com/jitsi/jitsi-meet/blob/f42772ec5bcc87ff6de17423d36df9bcad6e770d/doc/debian/jitsi-meet-prosody/prosody.cfg.lua-jvb.example#L57))
+- Create the visitors component in /etc/prosody/conf.d/jitmeet.example.com.cfg.lua:
+```
+Component "visitors.damencho.jitsi.net" "visitors_component"
```

Comment:
I'm a little confused by `jitmeet.example.com` and `damencho.jitsi.net`, but I guess it's not important